Source/WebCore/dom/TrustedTypePolicy.h

```diff
@@ -55,13 +57,17 @@ class TrustedTypePolicy : public ScriptWrappable, public RefCounted<TrustedTypeP
     ExceptionOr<String> getPolicyValue(TrustedType trustedTypeName, const String& input, FixedVector<JSC::Strong<JSC::Unknown>>&&, IfMissing = IfMissing::Throw);
     const String name() const { return m_name; }
 
+    const TrustedTypePolicyOptions& options() const { return m_options; }
```

Comment:
I think you want to add the `WTF_GUARDED_BY_LOCK` macro here to assist with static analysis.